### PR TITLE
Updated: Helm chart to v0.5.1 and fixed wrong repo url

### PR DESCRIPTION
**Pull request by GoliathLabs**
Hi,
I've updated the helm chart to fix the issues reported in #65

### CODE REVIEW

**Review by awlx — APPROVED** [review]
LGTM maybe check with @nodesocket

File: docs/index.yaml [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 entries:
   wg-access-server:
+  - apiVersion: v2
+    appVersion: v0.5.1
+    created: "2022-01-26T08:53:44.468726774Z"
+    description: A Wireguard VPN Access Server
+    digest: 2b3a8bbad6eaeffef433fe0bb0421b8005a3083abf1111afc898b61123749ad0
+    name: wg-access-server
+    urls:
+    - https://freie-netze.org/wg-access-server/charts/wg-access-server-v0.5.1.tgz
```
Comment (T0biii) [inline docs/index.yaml:11]: here i got a 404

Comment (GoliathLabs) [reply docs/index.yaml:11]: That's to be expected as github pages doesn't roll out non-master branches

Comment (T0biii) [reply docs/index.yaml:11]: ok i thought because the Version is already as release there: https://github.com/freifunkMUC/wg-access-server/releases

Comment (GoliathLabs) [reply docs/index.yaml:11]: Yeah, but we haven't bumped the chart version yet. That's why you're getting a 404

**Comment by DasSkelett** [comment]
How have those files been generated? Can you put the commands you've run into the PR description, so we can remember them in the future?

**Comment by GoliathLabs** [comment]
@DasSkelett done

**Pull request merged**